### DISCUSSION TITLE
Importing mailbox files

### DISCUSSION THREAD

**Discussion by dan-burt** [the-djmaze/snappymail#829] [Q&A]
Is it possible to import a local / archived MBOX (macOS) or OST / PST (Outlook) file(s)?

My intended use case is to offload my online mail host as it is now 10GB+ and the search through available clients is pathetic. So I was thinking of archiving everything before a reasonable date (like 3 or 4 years back) and "dump" it into a different mail client or app that can search the MBOX structure. This MBOX would need to be able to import more emails as I repeat this archive process.

I do not want to use it as my "active" mail client.

By placing it on my Nextcloud instance, I would be able to access it globally via a browser.

An example would be to search for a particular email in my "active" mail client and this "archive" client when I want to find things.

**Comment by cm-schl** [comment]
Hi,

this would only be possible if you have an other mail host where you could put your mails - from your description I think you have only one mail account (the one you would like to clean up). 
SnappyMail is a webmail client that needs to connect to an IMAP server where your mails are saved - without that you cannot use SnappyMail (see also https://github.com/the-djmaze/snappymail/wiki/Admin-Manual#what-snappymail-is-and-what-not).
